src/workerd/util/sqlite.h

```diff
@@ -43,7 +52,12 @@ class SqliteDatabase {
     uint64_t statementCount;
   };
 
-  SqliteDatabase(const Vfs& vfs, kj::Path path, kj::Maybe<kj::WriteMode> maybeMode = kj::none);
+  // Whoever creates the database, is required to make sure the SqliteObserver is alive at least
```

Comment:
This comment is stating something that [KJ style already requires by default](https://github.com/capnproto/capnproto/blob/v2/style-guide.md#ownership):

> A pointer or reference passed as a constructor parameter must remain valid for the lifetime of the constructed object.

So I think the comment can be omitted.

In any case, the same requirement applies to the existing `vfs` argument, so if the comment is going to be explicit about lifetimes it should mention that too.

Reply:
I was wondering the same thing, I added the comment in response to James' question regarding the same.
If it is part of the style guide, then it makes sense to remove the comment